### PR TITLE
#161804596 User should be able to see a homepage

### PR DESCRIPTION
**Pull request by kampkelly**
What does this PR do?
This PR adds a homepage to the website

Description of Task to be completed?
Show a homepage on going to the website url
#### How should this be manually tested?
Go to the root url, you should see a homepage showing different articles
#### Any background context you want to provide?
None
### Screenshots
phlox frontend homepage

#### What are the relevant pivotal tracker stories?
#161804596

### CODE REVIEW

File: src/components/ArticleCard/ArticleCard.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+import {
+  Card, CardTitle
+} from 'react-materialize';
+import './ArticleCard.scss';
+
+let articlePic = '';
+const history = createBrowserHistory({ forceRefresh: true });
+const linkTo = (path) => {
+  history.push(path);
+};
```
Comment (kampkelly) [inline src/components/ArticleCard/ArticleCard.js]: This is used to push to another route when a link is clicked.

File: src/components/CarouselSlider/CarouselSlider.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+export const CarouselSlider = props => (
+  <div>
+    <Carousel options={{ fullWidth: true, indicators: true }}>
+      { props.articles.map((article, index) => {// eslint-disable-line
+        if (!article.imgUrl) {
+          articlePic = 'https://via.placeholder.com/300?text=AuthorsHaven';
+        } else {
+          articlePic = article.imgUrl;
+        }
```
Comment (kampkelly) [inline src/components/CarouselSlider/CarouselSlider.js:19]: this sets a default placeholder image if there is no featured image for an article

File: src/containers/Home/Home.js [hunk 3]
```diff
@@ -11,6 +25,148 @@ class Home extends Component {
...
+   * @param {object} state
+   * @memberof Header
+   */
+  static getDerivedStateFromProps(props, state) {
+    if (document.querySelector('.carousel')) {
+      const elem = document.querySelector('.carousel');
+      const instance = M.Carousel.getInstance(elem);
+      setInterval(() => {
+        instance.next();
+      }, 5000);
```
Comment (kampkelly) [inline src/containers/Home/Home.js:58]: this slides the carousel showing the editor picks after every 5 minutes

File: src/styles/_variables.scss [hunk 4]
```diff
@@ -0,0 +1,4 @@
+$mobile: "(max-width: 600px) and (max-width: 600px)";
+$tablet: "(min-width: 600px) and (max-width: 992px)";
+$desktop: "(min-width: 992px) and (max-width: 1200px)";
+$large_desktop: "(min-width: 1200px)";
```
Comment (kampkelly) [inline src/styles/_variables.scss:6]: this contains variables that can be re-used in the application

File: src/App.scss [hunk 5]
```diff
@@ -3,3 +3,6 @@
   margin: 0;
   box-sizing: border-box;
 }
+body {
+  font-size: 18px;
+}
```
Comment (s-akinrele) [inline src/App.scss]: nitpick: new line

File: src/constants/Constants.js [hunk 6]
```diff
@@ -19,7 +21,7 @@ export const loginConstant = {
 };
 
 export const categoryConstant = {
-  GET_ALL_CATEGORY_URL: `${BASE_URL}/api/v1/categories`
+  GET_ALL_CATEGORY_URL: `${BASE_URL}/api/v1/categories`,
```
Comment (s-akinrele) [inline src/constants/Constants.js]: why the comma?

File: src/containers/Header/Header.scss [hunk 7]
```diff
@@ -1,14 +1,17 @@
-
-$primary-color: #06576D;
+@import '../../styles/variables';
+// $themeColor: #06576D;
+// $themeColor: #3B8CA1;
```
Comment (s-akinrele) [inline src/containers/Header/Header.scss]: remove commented code

File: src/styles/_variables.scss [hunk 8]
```diff
@@ -0,0 +1,7 @@
+$themeColor: #3B8CA1;
+// $themeColor: #06576D;
```
Comment (s-akinrele) [inline src/styles/_variables.scss]: remove commented code

**Review by s-akinrele — CHANGES_REQUESTED** [review]
I had some comments

**Review by s-akinrele — APPROVED** [review]
@kampkelly I dont like the idea that you are using `<a>` in some places and `<Link>` in other place
consistency is very important. I am going to go ahead to merge this PR, because I want you to have something to demo today. Take note!

**Pull request merged**